### DISCUSSION TITLE
Facing issue with customized color implementation in charting library, plz guide how to change it from default setting in my webpage

### DISCUSSION THREAD

**Discussion by bitrozex** [tradingview/lightweight-charts#788] [Q&A]
Plz guide me how to implement charting background customized color for my webpage.

**Comment by timocov** [comment]
Can you provide more details please?

**Reply by timocov** [reply]
@bitrozex Excuse me?

**Reply by bitrozex** [reply]
Hi, I wish to change the default color of the background screen of charting section( backside whole screen of candles). By default it's in two color whiz is white and black. I wish to customize.plz guide me how to change it n from where . Thank you 😊 I m so sorry for late reply ☺️

**Reply by timocov** [reply]
See https://github.com/tradingview/lightweight-charts/blob/master/docs/customization.md#chart-layout-options

**Comment by EvgeniyOrishko** [comment]
By default, the background type is 'gradient' and changing the background won't help. You also need to change backgroundType.
Or if you need a gradient see commented lines.
   overrides: {
        'paneProperties.background': 'rgba(32, 32, 32, 1)',
        'paneProperties.backgroundType': 'solid',
        
        //'paneProperties.backgroundType': 'gradient',
        //'paneProperties.backgroundGradientEndColor': 'rgb(188,7,34)',
        //'paneProperties.backgroundGradientStartColor': 'rgb(188,7,34)',
}

**Comment by le-el** [comment]
![image](https://github.com/user-attachments/assets/3d0ab22b-ddf8-4156-8983-934bc0db85cf)
I change the background color like this:
```
overrides: {
          'paneProperties.background': 'rgba(32, 32, 32, 1)',
          'paneProperties.backgroundType': 'solid',
        },
```

But I still get this blue background. What is the reason?
Please help me.
